### PR TITLE
Modify token language for teleport-kube-agent to allow for dynamic tokens

### PR DESCRIPTION
**Pull request by stevenGravy**
To match our Kubernetes multiple cluster instructions, adding that you can use a dynamic token with a set ttl with a long time.

### CODE REVIEW

**Review by awly — APPROVED** [review]

File: examples/chart/teleport-kube-agent/README.md [hunk 1]
```diff
@@ -12,10 +12,8 @@ To use it, you will need:
 - a reachable reverse tunnel port on the proxy (e.g. `teleport.example.com:3024`). The address is automatically retrieved from the Teleport proxy configuration.
 - a [static join
   token](https://goteleport.com/teleport/docs/admin-guide/#adding-nodes-to-the-cluster)
-  for this Teleport cluster (`$JOIN_TOKEN`)
-  - this chart does not currently support dynamic join tokens; please [file an
-    issue](https://github.com/gravitational/teleport/issues/new?labels=type%3A+feature+request&template=feature_request.md)
-    if you require support for dynamic tokens
+  for this Teleport cluster (`$JOIN_TOKEN`) or you can use a [dynamic join token](https://goteleport.com/docs/admin-guide/#short-lived-dynamic-tokens) with a lengthy time to live (ttl) (`tctl tokens add --type=kube --ttl=10000`).  The token setting will need updating if the static token is changed or before the dynamic token ttl expiration.
```
Comment (webvictim) [inline examples/chart/teleport-kube-agent/README.md:15]:
```suggestion
  for this Teleport cluster (`$JOIN_TOKEN`)
    - you can also use a [dynamic join token](https://goteleport.com/docs/admin-guide/#short-lived-dynamic-tokens) with a lengthy time to live (TTL) (`tctl tokens add --type=kube --ttl=10000`). The token setting will need updating if the token is changed or the TTL expires.
```

**Review by webvictim — APPROVED** [review]

**Comment by stevenGravy** [comment]
@russjones bot

**Review by klizhentas — CHANGES_REQUESTED** [review]
I would like to avoid recommending long-lived tokens and TTLs in our docs

**Comment by awly** [comment]
@klizhentas is right.
A dynamic token with a long TTL is basically a static token.

**Comment by webvictim** [comment]
A dynamic token with a fixed TTL is slightly more secure than a static token, because even a dynamic token with a 10,000 hour (416 day) TTL will _eventually_ expire, whereas a static token will not. I agree that we shouldn’t recommend the use of long-lived dynamic tokens though. 

The real way to fix this is to add support for Teleport to store its certs via Kubernetes API (https://github.com/gravitational/teleport/issues/4832)

**Comment by russjones** [comment]
Once we support dynamic tokens, we'll update the language at the same time. Closing for now.

**Pull request closed without merging**